### PR TITLE
Expand the docs for labelling tasks

### PR DESCRIPTION
**Pull request by CactiStaccingCrane**
I expanded the docs to include specific advice on how to input label data

### CODE REVIEW

File: docs/docs/tasks/label_assistant_reply.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # Label Assistant Reply
 
-Given the following discussion, provide labels for the final prompt.
+Given the following discussion, provide labels for the final prompt. In addition
+to the
+[Prompting Guide](https://projects.laion.ai/Open-Assistant/docs/guides/prompting),
+here are specific advices on how to correctly label the reply:
+
+- "Is the message spam?" is pretty self-explainatory.
```
Comment (AbdBarho) [inline docs/docs/tasks/label_assistant_reply.md:8]: not really sure if we should say "it is self explaintory" on the docs page.

**Review by AbdBarho — COMMENTED** [review]
Thank  you

File: docs/docs/tasks/label_assistant_reply.md (hunk 1 above)
Comment (CactiStaccingCrane) [reply docs/docs/tasks/label_assistant_reply.md]: Done, clarified on what is spam in both cases

**Comment by CactiStaccingCrane** [comment]
I'm just gonna close the pull request for now

**Comment by andrewm4894** [comment]
feel free to just set it to draft if you want and i can help with the pre-commit stuff if needed. I think we do need to build out the docs here more so make sense to add stuff like this.

**Pull request closed without merging**